### ISSUE TITLE
Incompatible with TypeScript 4.4

### ISSUE THREAD

**Issue by ffflorian** [PeculiarVentures/webcrypto-core#38]
When using `webcrypto-core` with TypeScript 4.4, I get the following errors:

```
node_modules/webcrypto-core/build/types/aes/cmac.d.ts:6:37 - error TS2304: Cannot find name 'AesCmacParams'.

6     checkAlgorithmParams(algorithm: AesCmacParams): void;
                                      ~~~~~~~~~~~~~

node_modules/webcrypto-core/build/types/aes/cmac.d.ts:7:32 - error TS2304: Cannot find name 'AesCmacParams'.

7     abstract onSign(algorithm: AesCmacParams, key: CryptoKey, data: ArrayBuffer, ...args: any[]): Promise<ArrayBuffer>;
                                 ~~~~~~~~~~~~~

node_modules/webcrypto-core/build/types/aes/cmac.d.ts:8:34 - error TS2304: Cannot find name 'AesCmacParams'.

8     abstract onVerify(algorithm: AesCmacParams, key: CryptoKey, signature: ArrayBuffer, data: ArrayBuffer, ...args: any[]): Promise<boolean>;
```

In my `tsconfig.json`, the required libs are added already:
```
...
    "lib": ["dom", "esnext"],
...
```

**Comment by microshine** [comment]
Fixed in `webcrypto-core@1.2.1`. Please try it

**Comment by ffflorian** [comment]
It works, thanks!
